digitaltwin/modelrepo/CommonExtensionMethods.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace Microsoft.Azure.DigitalTwin.Model.Service
+{
+    internal static class CommonExtensionMethods
+    {
+        private const char ValuePairDelimiter = ';';
```

Comment:
I think you want "KeyValuePair" everywhere

Reply:
Do you mean the naming of this can be KeyValuePairDelimiter ?